docs/styles/box_sizing.md

```diff
@@ -0,0 +1,42 @@
+# Box-sizing
```

Comment:
Struggled to follow this intro section. I don't have any suggestions for rewording at the moment, but just thought I'd note it here as something to potentially review. (Although I do think this concept is 100x easier to explain visually anyway)